examples/data-transfer-mssql/main.py

```diff
@@ -55,7 +56,7 @@ def connect_to_databases():
     return source_db_conn, dest_db_conn
 
 
-def get_connection(db_host, db_name, db_username, db_password):
+def get_connection(db_host, db_name, db_username, db_password) -> IO:
```

Comment:
The args - they need types too.
```suggestion
def get_connection(db_host: str, db_name: str, db_username: str, db_password: str) -> IO:
```